e2e/longrunning/simple_test.go

```diff
@@ -0,0 +1,82 @@
+package longrunning
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/knuu/pkg/instance"
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+const (
+	alpineImage = "alpine:3.20.3"
+	testTimeout = time.Minute * 5
+)
+
+func TestSimple(t *testing.T) {
```

Comment:
Can we add a test where we test to change something (for example, the image) of an already running instance

Reply:
yes will add it

Reply:
yes it is added now. There was an issue with pvc applying and that one is also resolved in the recent commit